workflow-templates/im-deploy-iis-website.yml

```diff
@@ -408,7 +400,7 @@ jobs:
         if: always()
         uses: im-open/post-status-to-teams-action@v1.1.4
         with:
-          title: ${{ needs.set-vars.outputs.WEBSITE_NAME }} Deployment to ${{ needs.set-vars.outputs.IIS_SERVER }}
+          title: ${{ env.WEBSITE_NAME }} Deployment to ${{ env.IIS_SERVER }}
```

Comment:
Since you've moved these variables into the deploy-site job you'll need to export them there so they are available as env variables here.

Reply:
`im-open/set-environment-variables-by-scope` sets `env` vars by default.  These are available in the same job.

Reply:
This step is in the `update-deployment-board-and-send-teams-notification` job.  These env vars are set in the `deploy-site` job.